merkletree/str.go

```diff
@@ -79,3 +79,15 @@ func (str *SignedTreeRoot) PreviousSTRHash() []byte {
 func (str *SignedTreeRoot) Signature() []byte {
 	return str.sig
 }
+
+func (str *SignedTreeRoot) LibVersion() []byte {
```

Comment:
Shouldn't ``LibVersion()``, ``HashID()`` and ``EpochDeadlinePolicy()`` be in ``policy.go``?